include/quda_milc_interface.h

```diff
@@ -728,6 +728,48 @@ extern "C" {
   void qudaGaugeForcePhased(int precision, int num_loop_types, double milc_loop_coeff[3], double eb3,
                             QudaMILCSiteArg_t *arg, int phase_in);
 
+  /**
+   * Compute the real traces of gauge loops, with direct application to computing the gauge
+   * action.  All fields here are CPU fields in MILC order, and their precisions should
+   * match.
+   *
+   * @param precision The precision of the field (2 - double, 1 - single)
```

Comment:
Missing `[in]` / `[out]` specifiers for the Doxygen parameters in this file.

Reply:
https://github.com/lattice/quda/pull/1308/commits/59cf5f0abf8e0c5cb25ef657c81d520720b48439